stack.yaml

```diff
@@ -21,6 +21,9 @@ extra-deps:
 - template-0.2.0.10@sha256:f822de4d34c45bc84b33a61bc112c15fedee6fa6dc414c62b10456395a868f85
 - inline-c-0.9.0.0
 - inline-c-cpp-0.4.0.0
+- Chart-cairo-1.9.1
+- cairo-0.13.8.0
+- gtk2hs-buildtools-0.13.8.0
```

Comment:
Feels a bit extreme to add those to the main config along with critical dependencies like `inline-c`. They're only used for one example, so maybe there's a way to separate them somehow?

Reply:
How about using a comment?
BTW, these are not needed now.

Reply:
Comment makes sense, unless there's a programatic way to select dependencies.